javascript/javascript2/week2/newHyfBay/index.js

```diff
@@ -0,0 +1,87 @@
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+const inputName = document.querySelector(".search #myInputName");
+const inputPrice = document.querySelector(".table-view-control #myInputPrice");
+const sortBy = document.querySelector(".sort select");

```

Comment:
Better to use id's instead of class. Normally for DOM elements we use id's and CSS we use class